packages/material-ui/src/styles/createSpacing.js

```diff
@@ -32,7 +32,7 @@ export default function createSpacing(spacingInput = 8) {
   }
 
   const spacing = (...args) => {
-    warning(args.length <= 4, `Too many arguments, expected between 1 and 4, got ${args.length}`);
+    warning(args.length <= 4, `Too many arguments, expected between 0 and 4, got ${args.length}`);
```

Comment:
I’m wondering whether to add the word provided i.e. ‘Too many arguments provided,’ would make this a suggestion but on mobile 😔

Reply:
```suggestion
    warning(args.length <= 4, `Material-UI: Too many arguments provided, expected between 0 and 4, got ${args.length}`);
```

Reply:
@eps1lon Thanks 😁

Reply:
You can type them on mobile as well. They're just normal codefences with `suggestion` as the language. I totally expect you to be able to write code without syntax errors on mobile :smiling_imp: